lib/dependencies/HarmonyImportSpecifierDependency.js

```diff
@@ -183,8 +184,41 @@ class HarmonyImportSpecifierDependency extends HarmonyImportDependency {
 			hash.update(stringifyUsedExports);
 		}
 	}
+
+	serialize(context) {
+		const { write } = context;
+		write(this.id);
+		write(this.name);
+		write(this.range);
+		write(this.strictExportPresence);
+		write(this.namespaceObjectAsContext);
+		write(this.callArgs);
```

Comment:
After Seans [tweet](https://twitter.com/TheLarkInn/status/1049747367782625280) I decided to try it out on our codebase. I found out, that cache didn't work for most of our modules. After debugging it I found out that this line is the most likely reason. There are no serializers defined for AST nodes so this will fail with "No serializer defined for Node" if `callArgs` are not empty. I've set up a reproduction case [here](https://github.com/SevInf/webpack-5-serializer).

Reply:
thanks for testing, this issue should be fixed now.

Note that this PR is a early prototype and many things are still missing. Only a few serializer has been added. Keep reporting the `No serializer defined for xxx` messages and I can add them (best with stack trace in `loglevel: "debug"`.